x-pack/plugins/observability_ai_assistant/public/components/chat/welcome_message_knowledge_base.tsx

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { noop } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPopover,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import usePrevious from 'react-use/lib/usePrevious';
+import useTimeoutFn from 'react-use/lib/useTimeoutFn';
+import useInterval from 'react-use/lib/useInterval';
+import { WelcomeMessageKnowledgeBaseSetupErrorPanel } from './welcome_message_knowledge_base_setup_error_panel';
+import type { UseKnowledgeBaseResult } from '../../hooks/use_knowledge_base';
+import type { UseGenAIConnectorsResult } from '../../hooks/use_genai_connectors';
+
+export function WelcomeMessageKnowledgeBase({
+  connectors,
+  knowledgeBase,
+}: {
+  connectors: UseGenAIConnectorsResult;
+  knowledgeBase: UseKnowledgeBaseResult;
+}) {
+  const previouslyNotInstalled = usePrevious(knowledgeBase.status.value?.ready === false);
+  const [showHasBeenInstalled, setShowHasBeenInstalled] = useState(false);
+  const [timeoutTime, setTimeoutTime] = useState(0);
+  const [, , reset] = useTimeoutFn(() => setShowHasBeenInstalled(false), timeoutTime);
```

Comment:
Nit: `reset` makes less sense to me since we use it here to start the timer? I'm not sure what would be a better name though